### PR TITLE
FAQ

### PR DESCRIPTION
**Pull request by panjenny0**
As per #5 
### Summary of changes
- Done frontend for displaying questions
- Done integration with modal component
- CRUD operations working
Screen Shot 2020-08-05 at 3 56 24 PM

### TODOs (prioritized from most to least important)
- [x] Remove question from frontend once it has been removed from firebase
- [x] time: x hours off for some reason
- [x] get FAQs by hackathon IDs
- [x] Enum support for question categories
- [ ] Add option to sort questions by categories
- [ ] Add invalid checks for all fields before submitting 
- [ ] have multi selects for hackathons a FAQ should apply to
- [x] Add confirmation for deleting question (use VIEW modal)

### CODE REVIEW

**Comment by panjenny0** [comment]
FYI still fixing linting errors

File: pages/faq.js [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+        .filter((id) => {
+          return id !== faqID;
+        })
+        .reduce((obj, id) => {
+          obj[id] = faqs[id];
+          return obj;
+        }, {})
+    );
+    handleClose();
+  };
```
[the remark this answers is not in the document]
Comment (panjenny0) [reply pages/faq.js]: This is resolved. Added a confirmation modal for user to confirm that they'd like to proceed with deleting a FAQ.

File: pages/faq.js [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+        <TableData actions>
+          <ActionsButtonContainer>
+            <Button
+              type={VIEW}
+              color={COLOR.TRANSPARENT}
+              onClick={() => setFaqViewing(faqs[props.faqID])}
+            />
+          </ActionsButtonContainer>
+          <ActionsButtonContainer>
+            <Link
```
[the remark this answers is not in the document]
Comment (panjenny0) [reply pages/faq.js]: I've removed the link here and references to `router`. My changes are ready to be reviewed now.

**Comment by Ciuffi** [comment]
@panjenny0 I fixed the linting errors for you

**Comment by panjenny0** [comment]
@Ciuffi Thanks! :) should I merge this?

**Comment by Ciuffi** [comment]
lets wait for one more review

@nwplus/internal-dev

**Review by yungalyx — APPROVED** [review]

**Pull request merged**